packages/core/admin/admin/src/content-manager/components/DynamicZone/components/Component/index.js

```diff
@@ -148,12 +150,20 @@ const Component = ({
           <AccordionContent>
             <AccordionContentRadius background="neutral0">
               <FocusTrap onEscape={() => onToggle(index)}>
-                <FieldComponent
-                  componentUid={componentUid}
-                  icon={icon}
-                  name={`${name}.${index}`}
-                  isFromDynamicZone
-                />
+                <Suspense
+                  fallback={
+                    <Flex justifyContent="center" paddingTop={4} paddingBottom={4}>
+                      <Loader />
+                    </Flex>
```

Comment:
is this the same for the other suspense fallback? If should so should we create a dedicated component?

Reply:
It's not the same the other is full-page (LoadingIndicatorPage), this one is inline